src/components/SuperDashListCard/SuperDashListCard.tsx

```diff
@@ -23,6 +23,7 @@ function SuperDashListCard(props: SuperDashListCardProps): JSX.Element {
 
   function handleClick() {
     const url = '/orgdash/id=' + props.id;
+    // Please endeavor to NOT remove both the window.location.replace(url) and the history.push(url) as both are very important for routing correctly.
```

Comment:
1. Put a warning for each line separately. Explain what will happen if it is removed.
1. This will help not just the coder but also the code reviewers who will see the comment removed.

Reply:
okay, on it

Reply:
Done @palisadoes